packages/lib/passport-interface/src/CredentialManager.ts

```diff
@@ -51,7 +53,7 @@ export function createCredentialCache(): CredentialCache {
 
 // Creates an in-memory cache with a TTL of one hour, backed by localStorage
 export function createStorageBackedCredentialCache(): CredentialCache {
-  return new StorageBackedMap("credential-cache");
+  return new StorageBackedMap("credential-cache-multi-email");
```

Comment:
Should we make an effort to clean up the old cache?
I assume the TTL is enforced in our code, so it's not going to get cleaned up automatically when it's abandoned.

Reply:
done
<img width="654" alt="Screenshot 2024-08-14 at 10 42 20 PM" src="https://github.com/user-attachments/assets/90eaba4b-a561-40f9-8646-19b8b1e4c433">